LICENSE

```diff
@@ -0,0 +1 @@
+TODO: Add your license here.
```

Comment:
Add a LICENSE here, you can use this one for example: https://github.com/flame-engine/flame_tiled/blob/master/LICENSE